cns/middlewares/k8sSwiftV2_windows.go

```diff
@@ -16,10 +32,16 @@ func (k *K8sSWIFTv2Middleware) setRoutes(podIPInfo *cns.PodIpInfo) error {
 		// TODO: Remove this once HNS fix is ready
 		route := cns.Route{
 			IPAddress:        "0.0.0.0/0",
-			GatewayIPAddress: "0.0.0.0",
+			GatewayIPAddress: defaultGateway,
```

Comment:
I believe your other PR had some addDefaultRoute-- what happened to it here-- is it not used?